examples/all-clusters-minimal-app/all-clusters-common/all-clusters-minimal-app.zap

```diff
@@ -1524,6 +1540,36 @@
               "maxInterval": 65344,
               "reportableChange": 0
             }
+          ],
+          "events": [
+            {
+              "name": "StartUp",
+              "code": 0,
+              "mfgCode": null,
+              "side": "server",
+              "included": 1
+            },
+            {
+              "name": "ShutDown",
```

Comment:
This event is optional.  The minimal app is not supposed to enable any optional features.  I know you were just going through this by hand and it was a pain, but can you please disable all the non-mandatory events in this specific app?

Reply:
Updated this minimal app to support only Required (mandatory) events and disable all optional events.